rna/rna-collect-multiple-metrics/rna-collect-multiple-metrics.inputs.json

```diff
@@ -0,0 +1,20 @@
+{
+  "##_COMMENT1": "Set your per-sample values: name and BAM path, and intervals file path.",
+  "RNACollectMultipleMetrics.sample_name": "__INSERT_SAMPLE_NAME__",
+  "RNACollectMultipleMetrics.bam_file": "__INSERT_GCS_PATH_TO_BAM_FILE__",
+
+  "##_COMMENT2": "Path to reference FASTA file.",
+  "RNACollectMultipleMetrics.reference_sequence_file": "__INSERT_GCS_PATH_TO_REF_FASTA_FILE__",
```

Comment:
For the other workflows in this repo, we included the path to the reference file. For this one it should be:

```
gs://amp-pd-transcriptomics/inputs/reference/gencode/GRCh38.primary_assembly.genome.fa.gz
```